ui/src/app/settings/components/project-details/resource-lists-panel.tsx

```diff
@@ -203,33 +215,44 @@ function viewDestinationServiceAccountsInfoList(type: field, proj: Project) {
 
 function editList(type: field, formApi: FormApi) {
     const info = infoByField[type];
+    const isClusterResourceList = type === 'clusterResourceWhitelist' || type === 'clusterResourceBlacklist';
 
     return (
         <React.Fragment>
             <p className='project-details__list-title'>
                 {info.title} {helpTip(info.helpText)}
             </p>
             <div className='row white-box__details-row'>
-                <div className='columns small-4'>Kind</div>
-                <div className='columns small-8'>Group</div>
+                <div className={isClusterResourceList ? 'columns small-4' : 'columns small-4'}>Kind</div>
+                <div className={isClusterResourceList ? 'columns small-4' : 'columns small-8'}>Group</div>
+                {isClusterResourceList && <div className='columns small-4'>Name (optional)</div>}
             </div>
             {(formApi.values.spec[type] || []).map((_: Project, i: number) => (
                 <div className='row white-box__details-row' key={i}>
-                    <div className='columns small-4'>
+                    <div className={isClusterResourceList ? 'columns small-4' : 'columns small-4'}>
```

Comment:
typo so the class is the same as the div below?
```suggestion
                    <div className={isClusterResourceList ? 'columns small-4' : 'columns small-8'}>
```

Reply:
Fixed